franka_gazebo/src/joint.cpp

```diff
@@ -64,10 +64,10 @@ double Joint::getLinkMass() const {
 }
 
 bool Joint::isInCollision() const {
```

Comment:
@gollth, @marcbone Just a minor point, maybe you want to put a note in the documentation about how the simulated `cartesian_collision` and `joint_collision` differ from the real robot. I noticed that in the simulation, users do not need to send a reset command for the `<>_contact` variables to return to zero.

https://github.com/frankaemika/libfranka/blob/f1f46fb008a37eb0d1dba00c971ff7e5a7bfbfd3/include/franka/robot_state.h#L258-L274

https://github.com/frankaemika/libfranka/blob/f1f46fb008a37eb0d1dba00c971ff7e5a7bfbfd3/include/franka/robot_state.h#L258-L274

Reply:
This comment was added to #363.